### ISSUE TITLE
Add CoD info to the docs

### ISSUE THREAD

**Issue by ElPaisano** [ipfs/ipfs-docs#1436]
We received a request from the Bacalhau team to create a case study page similar to https://docs.ipfs.tech/case-studies/arbol/ for [Bacalhau](https://docs.bacalhau.org/)

**Comment by TMoMoreau** [comment]
If I'm not mistaken, and I may very well be, I believe the case studies are generally written by a member or representative of the project that the case study is about.

**Comment by ElPaisano** [comment]
@2color @mishmosh @TheDiscordian @TMoMoreau I know there was some discussion during [the reorg PR](https://github.com/ipfs/ipfs-docs/pull/1405) about including case studies and where they should be located.

Thoughts on including a case studies section long term? If we aren't including it long term, is it worth creating any more case studies? Either way, thoughts on creating this case study?

**Comment by ElPaisano** [comment]
> If I'm not mistaken, and I may very well be, I believe the case studies are generally written by a member or representative of the project that the case study is about.

Just saw this @TMoMoreau. Either way, wondering if we should be adding any more content to case studies, if we are going to be moving that content out of the docs

**Comment by TMoMoreau** [comment]
Seems like it's going to be written either way, so if it happens to live in the docs for even just a month before moving, I suppose it doesn't matter if we add it in.

**Comment by ElPaisano** [comment]
@2color any thoughts on adding case studies? I haven't been here long enough to know how we create case studies.

If the org requesting the case study generally writes it, I'd suggest we create a MD template - basically a .md file that has similar structure to the other case studies - and give it to them with some "fill in the blank instructions". 

I would note that this request was made awhile back, so I do think that we should consider adding it in ourselves as well.

**Comment by 2color** [comment]
Ultimately it's a question of who writes the case study. But generally, the case studies tend to be for actual products with end users where as Bacalhau is more of a developer tool -- and nonetheless an important one.

Given that, my suggestion would be to add a page to concepts about verifiable and distributed compute with content addressed data and introduce Bacalhau and maybe some other related projects such as IPVM.

**Comment by ElPaisano** [comment]
Created https://github.com/ipfs/ipfs-docs/pull/1714 for this.

TLDR
Added page in concepts about CoD that intros both Bacalahu and IPVM. Basic concepts are disucssed. In the case of bacalhau, also links out to hand-on getting started guide

**Comment by ElPaisano** [comment]
Closed by https://github.com/ipfs/ipfs-docs/pull/1697
